libtenzir/CMakeLists.txt

```diff
@@ -176,7 +176,7 @@ add_library(libtenzir ${libtenzir_sources})
 TenzirTargetEnableTooling(libtenzir)
 add_library(tenzir::libtenzir ALIAS libtenzir)
 
-add_library(libtenzir_builtins STATIC ${libtenzir_builtins_sources})
+add_library(libtenzir_builtins OBJECT ${libtenzir_builtins_sources})
```

Comment:
There's also the option of using the approach from https://github.com/tenzir/tenzir/pull/3754 (which was for reasons I still don't understand reverted in https://github.com/tenzir/tenzir/pull/3766). That has the advantage of also massively reducing linker time.

Reply:
I think the overall effect should be similar, no? No additional linking should take place here.